### PR TITLE
Embed responsively. Fix #12

### LINKED ISSUE #12: Responsive video embeds
http://embedresponsively.com

### PR DESCRIPTION
**Pull request by jpadilla**

### CODE REVIEW

**Review by froi — CHANGES_REQUESTED** [review]
Baje el branch y corriendolo local veo mucho whitespace entre el embed del podcast epidose y los enlaces. Serai bueno eliminar esto.

image

Probé en:

* Firefox: 61.0.1 (64-bit)
* Chrome: 67.0.3396.99 (Official Build) (64-bit)
* Safari: 11.1.2 (13605.3.8)

File: _podcast/2018-05-23-fullstack-nights-podcast-ep1.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ description: Estaremos hablando de los cambios en nuestras carreras en el pasado
 
 Estaremos hablando de los cambios en nuestras carreras en el pasado año y lo que pensamos sobre mentoría.
 
-<iframe src="https://anchor.fm/fullstack-nights-podcast/embed/episodes/Fullstack-Nights-Podcast---Episode-1-e1h4c1" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+<div class="embed-container">
```
Comment (froi) [inline _podcast/2018-05-23-fullstack-nights-podcast-ep1.md]: Shouldn't this be `embed-container.anchorfm` and not `embed-container`?

**Comment by jpadilla** [comment]
@froi just addressed that

File: _podcast/2018-06-12-fullstack-nights-podcast-ep2.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ En este episodio discutimos:
 * ¿Qué resposabilidades tenemos como developers?
 * Lecciones implementando el estandard de PCI (Payment Card Industry)
 
-<iframe src="https://anchor.fm/fullstack-nights-podcast/embed/episodes/Episodio-2---Responsabilidad-y-Seguridad-e1kei4" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+<div class="embed-container">
```
Comment (froi) [inline _podcast/2018-06-12-fullstack-nights-podcast-ep2.md]: Same question as above.

File: _podcast/2018-07-16-fullstack-nights-podcast-ep3.md [hunk 3]
```diff
@@ -17,8 +17,9 @@ En este episodio discutimos:
 * Proyectos en Code for Orlando
 * Seguridad digital para periodistas.
 
-
-<iframe src="https://anchor.fm/fullstack-nights-podcast/embed/episodes/Episodio-3---Code-for-Orlando-y-Seguridad-para-Periodistas-e1rms3" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+<div class="embed-container">
```
Comment (froi) [inline _podcast/2018-07-16-fullstack-nights-podcast-ep3.md]: Same question as above.

**Review by froi — COMMENTED** [review]
I think that `embed-container.anchorfm` is not being used in the podcast pages and I'm getting the same results.

File: _podcast/2018-05-23-fullstack-nights-podcast-ep1.md (hunk 1 above)
Comment (jpadilla) [reply _podcast/2018-05-23-fullstack-nights-podcast-ep1.md]: @froi yeah yeah missed that, sorry about that.

**Review by froi — APPROVED** [review]
Awesomeness.

Just tested locally. Thanks for getting this done

**Pull request merged**